developers/weaviate/configuration/authentication.md

```diff
@@ -256,6 +271,32 @@ $ curl http://localhost:8080/v1/objects -H "Authorization: Bearer {Bearer}"
 
 If using a Weaviate client library, click on the relevant link for [Python](../client-libraries/python.md#authentication), [Javascript](../client-libraries/javascript.md#authentication), [Java](../client-libraries/java.md#authentication) or [Go](../client-libraries/go.md#authentication) to find instructions on how to attach a token with that client.
 
+## Anonymous access
```

Comment:
Hmm yeah it's down pretty far. I think it would be good to lead with the WCS auth stuff, so I have added a note up top to say that you can run Weaviate with/without auth, and links to in-page anchors.